### DISCUSSION TITLE
Word by word diarization Whisper-Pyannote

### DISCUSSION THREAD

**Discussion by Jose-Sabater** [openai/whisper#1945] [Show and tell]
I was trying to use the great work from [yinruiqing](https://github.com/yinruiqing/pyannote-whisper) but was getting some problems when linking the transcripts with the diarization results.

I created a new , very simple way of doing this using the word by word timestamps from whisper. 
Tried multiple examples in different languages and worked quite well!
Check it out [Whisper-Pyannote](https://github.com/Jose-Sabater/whisper-pyannote)

**Comment by SIlver--** [comment]
Thanks! Really helps with making one's own diarization feature without having to rely directly on a library.
